tests/apps/stateapp/app.go

```diff
@@ -0,0 +1,226 @@
+	log.Printf("Processing get request for %s.", key)
+	url := strings.Join([]string { stateURL, key }, "/")
+	log.Printf("Fetching state from %s", url)
+	res, err := http.Get(url)
+	if err != nil {
+		return nil, fmt.Errorf("Could not get value for key %s from Dapr: %s", key, err.Error())
+	}
+
+	body, err := ioutil.ReadAll(res.Body)
+    if err != nil {
+        return nil, fmt.Errorf("Could not load value for key %s from Dapr: %s", key, err.Error())
+	}
+	
+	log.Printf("Found state for key %s: %s", key, body)
+
+	var state = new(appState)
+	if len(body) == 0 {
+		return nil, nil
+	}
+
+	// a key not found in Dapr will return 200 but an empty response.
+	err = json.Unmarshal([]byte(body), &state)
+	if err != nil {
+		return nil, fmt.Errorf("Could not parse value for key %s from Dapr: %s", key, err.Error())
+	}
+
+	return state, nil
+}
+
+func getAll(states []daprState) ([]daprState, error) {
+	log.Printf("Processing get request for %d states.", len(states))
+
+	var output []daprState
+	for _, state := range states {
+		value, err := get(state.Key)
+
+		if err != nil {
+			return nil, err
+		}
+
+		log.Printf("Result for get request for key %s: %v", state.Key, value)
+		output = append(output, daprState {
+			Key: state.Key,
+			Value: value,
+		})
+	}
+
+	log.Printf("Result for get request for %d states: %v", len(states), output)
+	return output, nil
+}
+
+func delete(key string) error {
+	log.Printf("Processing delete request for %s.", key)
+	url := strings.Join([]string { stateURL, key }, "/")
+	req, err := http.NewRequest("DELETE", url, nil)
+	if err != nil {
+		return fmt.Errorf("Could not create delete request for key %s in Dapr: %s", key, err.Error())
+	}
+
+	log.Printf("Deleting state for %s", url)
+	client := &http.Client{}
+	_, err = client.Do(req)
+	if err != nil {
+		return fmt.Errorf("Could not delete key %s in Dapr: %s", key, err.Error())
+	}
+
+	return nil
+}
+
+func deleteAll(states []daprState) error {
+	log.Printf("Processing delete request for %d states.", len(states))
+
+	for _, state := range states {
+		err := delete(state.Key)
+
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// handles all APIs
+func handler(w http.ResponseWriter, r *http.Request) {
+	log.Printf("Processing request for %s", r.URL.RequestURI())
+
+	// Retrieve request body contents
+	var req requestResponse
+	err := json.NewDecoder(r.Body).Decode(&req)
+	if err != nil {
+		log.Printf("Could not parse request body: %s", err.Error())
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(requestResponse {
+			Message: err.Error(),
+		})
+		return
+	}
+
+    var res = requestResponse {}
+	var uri = r.URL.RequestURI()
+	var states []daprState
+	var statusCode = http.StatusOK
+
+	res.StartTime = epoch()
+	
+	switch uri {
+	case "/test/save":
+		err = save(req.States)
+	case "/test/get":
+		states, err = getAll(req.States)
+		res.States = states
+	case "/test/delete":
+		err = deleteAll(req.States)
```

Comment:
Can we use the same way of hellodapr ? 

https://github.com/dapr/dapr/blob/f5087726114e55fdd7695125ef449b17e2877b53/tests/apps/hellodapr/app.go#L41

https://github.com/dapr/dapr/blob/f5087726114e55fdd7695125ef449b17e2877b53/tests/apps/hellodapr/app.go#L59-L64

Reply:
Done.